### DISCUSSION TITLE
Official Figma Support

### DISCUSSION THREAD

**Discussion by jbng** [ant-design/ant-design#40405] [Q&A] [answered]
Hi, 

I really enjoy using Ant design. It's a pity that there is no official support for Figma.
I was wondering if it's at least in discussion in the official design team.

By the way, this guy is doing well thanks to the lack of Figma support : 
https://www.antforfigma.com/pricing

Thanks

**Comment by arvinxx** [comment, marked as answer]
We won't support figma for now, and maybe never.  Because we don't use figma  internally so we have no people to work for figma version design resources. Maybe you can ask for figma community when to upgrade antd v5 design resources.

If you need design resources, you can try with [Kitchen](https://kitchen.alipay.com/)(a sketch plugin), which can dynamtic geneate antd components with custom theme. Here is [introduction](https://www.yuque.com/kitchen/changelog/3.2.0).

**Reply by jbng** [reply]
I understand, it's a pity but thanks for the clarification. 
Using figma might be a clue to get a lot more professional users, especially in Western countries.

**Reply by shawnim** [reply]
Totally agree. Many people in the US end up using Material Design because of better support in Figma.
We use antd v4 with Figma but want to use v5 features and are limited which results in pressure to change to Material.
